addons/mail/static/src/models/thread_view_topbar.js

```diff
@@ -399,7 +399,10 @@ registerModel({
                 }
                 return `/mail/channel/${this.thread.id}/guest/${this.messaging.currentGuest.id}/avatar_128?unique=${this.messaging.currentGuest.name}`;
             }
-            return this.messaging.currentPartner.avatarUrl;
+            if (this.messaging.currentPartner) {
+                return this.messaging.currentPartner.avatarUrl;
+            }
+            return '';
```

Comment:
as the thread view topbar is never displayed in the context of livechat it's strange you had to do this diff now

Reply:
I just guarded everything that was on `currentPartner`, as there is no `currentPartner` in frontend.
`currentUserGuest` doesn't work too, hence why guard.

All in all, it's too much effort to properly to everything at once.

The approach I take here is to have models available, even with few missing features (mostly due to special integration needed for frontend livechat specifically). When models are available and it's stable, then I can start moving a few LOCs at a time from legacy public livechat to models.